spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/data/cassandra/DataCassandraTestReactiveIntegrationTests.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.test.autoconfigure.data.cassandra;
+
+import com.datastax.driver.core.Session;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.cassandra.CassandraTestServer;
+import org.springframework.data.cassandra.core.ReactiveCassandraTemplate;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Sample tests for {@link DataCassandraTest} using reactive repositories.
+ *
+ * @author Eddú Meléndez
+ */
+@RunWith(SpringRunner.class)
+@DataCassandraTest
+@TestPropertySource(properties = {"spring.data.cassandra.schemaAction=recreate_drop_unused",
```

Comment:
Not sure I like the idea of having to define those. IMO, if we have to then users of the annotation will. And if they do, then we probably need an OOB property for this with a sensible value